Vagrantfile

```diff
@@ -18,5 +17,6 @@ Vagrant.configure("2") do |config|
   config.vm.provision "ansible_local" do |ansible|
     ansible.playbook = "ansible/playbook.yml"
     ansible.compatibility_mode = "2.0"
+    ansible.install_mode = "pip"
```

Comment:
The `default` install mode doesn't work anymore, it throws errors for missing packages.

Reply:
Yes, I had the same problem 👍